### PR TITLE
fix abnormal-security-get-employee-identity-analysis command

### PR DESCRIPTION
**Pull request by tyng94**
fix employee_identity_analysis endpoint

### CODE REVIEW

**Review by akemburu — COMMENTED** [review]
what is being fixed here? it looks like we are just re-ordering the commands?

File: Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py [hunk 1]
```diff
@@ -345,7 +345,7 @@ def get_employee_identity_analysis_genome_data_command(client, args):
     headers = ['description', 'key', 'name', 'values']
 
     markdown = tableToMarkdown(
-        f"Analysis of {email_address}", response.get('histograms', []), headers=headers, removeNull=True)
+        f"Analysis of {email_address}", response.get('data', []), headers=headers, removeNull=True)
```
Comment (tyng94) [inline Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:348]: @akemburu this one line change is the main issue. see https://github.com/abnormal-security/source/pull/13747

Comment (akemburu) [reply Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py:348]: thanks TY

**Review by akemburu — APPROVED** [review]
lgtm

**Pull request merged**